.github/workflows/w3c-publish.yml

```diff
@@ -43,10 +48,16 @@ jobs:
       - name: Setup Sphinx
         run: pip install six && pip install sphinx==5.1.0
       - name: Publish all specs to their https://www.w3.org/TR/ URLs
+        if: env.W3C_ECHIDNA_TOKEN_CORE
         run: cd document && make -e WD-echidna-CI
         env:
-          W3C_STATUS: ${{ github.event_name == 'push' && 'WD' || inputs.w3c-status }}
           W3C_ECHIDNA_TOKEN_CORE: ${{ secrets.W3C_ECHIDNA_TOKEN_CORE }}
           W3C_ECHIDNA_TOKEN_JSAPI: ${{ secrets.W3C_ECHIDNA_TOKEN_JSAPI }}
           W3C_ECHIDNA_TOKEN_WEBAPI: ${{ secrets.W3C_ECHIDNA_TOKEN_WEBAPI }}
-          YARN_ENABLE_IMMUTABLE_INSTALLS: false
+          ECHIDNA_DRYRUN: ${{ github.event_name == 'pull_request' && github.repository == 'WebAssembly/spec'}}
```

Comment:
I'm confused, why do a dry run when it is a PR against the main repo? Shouldn't it be the other way round? In fact, shouldn't it be a dry run unless it is an actual _push_ against the main repo?

Also, this flag is set for the WD-echidna-CI target, but in two of the makefiles it's only used by the WD-echidna target.

Reply:
Note though that as I mentioned yesterday, this doesn't actually work at all yet for PRs against upstream that originate from a fork, because those don't get repository secrets from either the source or the destination repo.